### PR TITLE
[ko] Translate ''managed-services"

### PR DESCRIPTION
**Pull request by yuri0329**
### Describe your changes
[ko] Translate ''managed-services"

### Related issue number or link (ex: `resolves #issue-number`)
Relevant with https://github.com/cncf/glossary/blob/main/content/en/managed-services.md

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ko/managed-services.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: 관리형 서비스 (Managed services)
+status: Completed
+category: 기술
+tags: ["", "", ""]
+---
+
+## 개념
+
+관리형 서비스(Managed services)는 운영 및 관리가 제3자에 의해 처리되는 소프트웨어 오퍼링이다.
```
Comment (yunkon-kim) [inline content/ko/managed-services.md:10]:
```suggestion
관리형 서비스(Managed services)는 제3자의 운영 및 관리를 바탕으로 제공되는 소프트웨어 제품이다. 
```

File: content/ko/managed-services.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: 관리형 서비스 (Managed services)
+status: Completed
+category: 기술
+tags: ["", "", ""]
+---
+
+## 개념
+
+관리형 서비스(Managed services)는 운영 및 관리가 제3자에 의해 처리되는 소프트웨어 오퍼링이다. 
+예를 들어, 아마존(Amazon)의 RDS와 같은 서비스형 데이터 베이스 또는 데이터독(Datadog)와 같은 외부 모니터링 서비스가 있다.
```
Comment (yunkon-kim) [inline content/ko/managed-services.md:11]:
```suggestion
예를 들어, 아마존(Amazon)의 RDS와 같은 서비스형 데이터베이스 제품 또는 데이터독(Datadog)와 같은 외부 모니터링 서비스가 있다. 
```

File: content/ko/managed-services.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+## 개념
+
+관리형 서비스(Managed services)는 운영 및 관리가 제3자에 의해 처리되는 소프트웨어 오퍼링이다. 
+예를 들어, 아마존(Amazon)의 RDS와 같은 서비스형 데이터 베이스 또는 데이터독(Datadog)와 같은 외부 모니터링 서비스가 있다. 
+
+## 다루는 문제
+
+특히 최신 스택(stack)을 구성하는 다양한 기술을 모두 고려할 때 소프트웨어 관리는 복잡하다.
```
Comment (yunkon-kim) [inline content/ko/managed-services.md:15]:
```suggestion
특히 최신 스택(stack)을 구성하는 다양한 기술을 모두 고려해야 하므로 소프트웨어 관리는 복잡하다.
```

File: content/ko/managed-services.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+각 측면을 관리하거나 사내 전문가를 두는 것은 비용이 너무 많이 들거나 엔지니어가 시간을 투자할 만큼의 가치가 없을 수 있다.
```
Comment (yunkon-kim) [inline content/ko/managed-services.md:16]:
```suggestion
이와 관련한 각각의 측면 관리 및/또는 사내 전문가 보유는 비용이 너무 많이 들거나 엔지니어의 시간을 낭비할 가치가 없을 수 있다.
```

File: content/ko/managed-services.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+팀은 쉽게 아웃소싱(outsourced)할 수 있는 운영 업무를 처리하는 것 보다 새로운 기능을 구축하는 것이 더 나을 것이다.
```
Comment (yunkon-kim) [inline content/ko/managed-services.md:17]:
```suggestion
팀은 쉽게 아웃소싱(outsourced)할 수 있는 운영 업무를 처리하는 것보다 새로운 기능을 구축하는 것이 더 나을 것이다.
```

File: content/ko/managed-services.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+## 문제 해결 방식
+
+관리형 서비스는 첫날부터 운영 오버헤드(overhead)가 거의 없이 사용할 수 있다.
+이를 통해 조직은 잘 정의되고, 일반적으로 [API](/application-programming-interface/) 중심의 경계로 핵심 역량을 벗어나는 작업을 효과적으로 아웃소싱(outsource)할 수 있다.
```
Comment (yunkon-kim) [inline content/ko/managed-services.md:22]:
```suggestion
이를 통해, 조직은 [API](/application-programming-interface/)와 같은 명확하게 정의된 경계를 바탕으로 핵심 역량을 벗어나는 작업을 효과적으로 아웃소싱(outsource)할 수 있다.
```

File: content/ko/managed-services.md (hunk 4 above)
Comment (yunkon-kim) [reply content/ko/managed-services.md:16]: 원문 한 문장이 다소 추상적이고, 많은 내용이 얽혀 있어 한글화가 쉽지 않은 것 같습니다... ^^:;

**Review by yunkon-kim — COMMENTED** [review]
@yuri0329 Deprecated된 용어라서 간단히 리뷰 의견 드렸습니다 ^^

**Review by yunkon-kim — APPROVED** [review]
@yuri0329 LGTM!

File: content/ko/managed-services.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+---
+title: 관리형 서비스 (Managed services)
+status: Deprecated
```
Comment (seokho-son) [inline content/ko/managed-services.md:3]:
```suggestion
status: Deprecated
draft: true
```

Comment (seokho-son) [reply content/ko/managed-services.md:3]: Deprecated 처리를 위해 draft: true 를 추가해두었습니다.

**Review by seokho-son — APPROVED** [review]
lgtm!

**Pull request merged**